frontend/amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -78,13 +99,15 @@ def create_issue(self,
                      owner_ids: List[str],
                      frequent_user_ids: List[str],
                      priority_level: str,
+                     project_key: str,
```

Comment:
I found that since the issue clients inherit from a base class it expected all the signatures to match, but I ended up changing to use kwargs to remove the need to explicitly include the ones that aren't used by both clients which makes it a bit cleaner